test/functional/test-url-replacements.js

```diff
@@ -1646,19 +1648,21 @@ describes.sandboxed('UrlReplacements', {}, () => {
           'https://example2.com/link?out=QUERY_PARAM(foo)');
     });
 
-    it('should replace CID', () => {
+    it('should replace whitelisted fields', () => {
```

Comment:
I feel we can have a test for "should not replace non whitelist field". not in this one though. lgtm

Reply:
we have one here: https://github.com/ampproject/amphtml/blob/master/test/functional/test-url-replacements.js#L1604